spec/default_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'spec_helper'
+
+describe 'osl-app::default' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(CENTOS_7).converge('sudo', described_recipe)
+  end
+  include_context 'common_stubs'
+  it do
```

Comment:
Our new standard is to let chefspec do this automatically unless we want to have an explicit message. As it stands now, here's what it prints:

```
 osl-app::default
  should not raise Exception
```